gtp2ogs.js

```diff
@@ -2226,9 +2248,9 @@ function num2gtpchar(num) { /* {{{ */
     return "abcdefghjklmnopqrstuvwxyz"[num];
 } /* }}} */
 function rank2str(r) { /* {{{ */
```

Comment:
I suggest we name this RankToString

Reply:
yes we can do it, but i dont want to rebase endlessly lol
so i can commit from website if needed, once all changes have been discussed :+1:

Reply:
A minor change like renaming a function doesn't need rebasing. If you make the change to your branch and push the commit it'll automatically update the PR. Unless I don't understand what you meant.

Reply:
@roy7 

i meant i dont want to commit until i made all wanted changes, if i commit change function name then later there are other things to commit then i'll want to rebase, its what i meant :)